File: api/code-samples/python/eth/block/getHeaderByHash.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xb8deae63002d2b6aa33247c8ef545383ee0fd2282ac9b49dbbb74114389ddb5c"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_header_by_hash(blockHash)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getHeaderByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockHash = '0xba647d41423faeebe8a7c64737d284fc2eba6f0388a3e1ebf6243db509ec1577'
    
    sdk.eth.getHeaderByHash(blockHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()